ansible/roles/test/tasks/ptf_runner_reboot.yml

```diff
@@ -61,7 +61,7 @@
         - ports_file=\"/tmp/ports.json\"
         - dut_mac='{{ dut_mac }}'
         - dut_vlan_ip='192.168.0.1'
-        - default_ip_range='192.168.0.0/16'
+        - default_ip_range='192.168.100.0/18'
```

Comment:
can you change the fast-reboot script in pytest as well? @tamer-dev, is there a same problem for the pytest version?

Reply:
@lguohan same fix for `master` https://github.com/Azure/sonic-mgmt/pull/1637

Reply:
@lguohan yes, the pytest reboot test has prefix lent of 16. @nazariig can you please modify this file as well:

https://github.com/Azure/sonic-mgmt/blob/master/tests/common/fixtures/advanced_reboot.py#L129